zerver/lib/actions.py

```diff
@@ -4506,6 +4507,7 @@ def notify_avatar_url_change(user_profile: UserProfile) -> None:
         # Even clients using client_gravatar don't need the email,
         # since we're sending the URL anyway.
         user_id=user_profile.id,
+        email_address_visibility=user_profile.email_address_visibility,
```

Comment:
Sending `email_address_visibility` here to compute `client_gravatar` in `apply_event`.